opentelemetry-user-events-logs/src/logs/exporter.rs

```diff
@@ -52,7 +52,8 @@ pub(crate) struct UserEventsExporter {
 }
 
 const EVENT_ID: &str = "event_id";
-const EVENT_NAME: &str = "event_name";
+const EVENT_NAME_PRIMARY: &str = "event_name";
+const EVENT_NAME_SECONDARY: &str = "name";
```

Comment:
Maybe reverse primary and secondary?

Lets add a comment indicating why we need to look for attributes with either of these names.

Reply:
The `name` metadata field defaults to `file:line` if it is not set using event macros, or with older version of tracing. So, making it primary will not allow to use `event_name`.